src/nmodl/main.cpp

```diff
@@ -319,6 +322,9 @@ int main(int argc, const char* argv[]) {
     llvm_opt->add_flag("--ir",
         llvm_ir,
         "Generate LLVM IR ({})"_format(llvm_ir))->ignore_case();
+    llvm_opt->add_flag("--add-debug-info",
+                       add_debug_information,
+                       "Add debug information ({})"_format(add_debug_information))->ignore_case();
```

Comment:
Wondering if we should always add debug info (or have it ON)

Reply:
We can, yes, then a flag `--disable-debug-info` can be used to turn it off instead. What do you think?

Reply:
👍 : the reason I mentioned to make it ON is that during development we need this all the time. So we don't need to add one more flag to CLI. (at least for now)

Reply:
Agree! Will change that!